README.md

```diff
@@ -1,8 +1,48 @@
-# radis.app
+## 🌱 Radis App
 
-[![Production deployment](https://github.com/suzil/radis-app/actions/workflows/cd.yml/badge.svg)](https://github.com/suzil/radis-app/actions/workflows/cd.yml)
```

Comment:
It would be nice to keep the build badge so that it's easy to spot if deployment is failing.